.pre-commit-config.yaml

```diff
@@ -1,45 +1,45 @@
 default_language_version:
-    python: python3.8
```

Comment:
I'm a bit curious to know which formatting tool / editor picks each of these two formatting styles for yaml files. Maybe we need to add a pre-commit tool for fix this in the future

Reply:
for reference: it was formatted using prettier which I have as default in vscode